src/utils/transaction.ts

```diff
@@ -66,7 +74,10 @@ export const fromCallsToExecuteCalldata_cairo1 = (calls: Call[]) => {
   const orderCalls = calls.map((call) => ({
     contractAddress: call.contractAddress,
     entrypoint: call.entrypoint,
-    calldata: call.calldata,
+    calldata:
+      Array.isArray(call.calldata) && '__compiled' in call.calldata
```

Comment:
```suggestion
      Array.isArray(call.calldata) && '__compiled__' in call.calldata
```

The compilation flag check is missing two underscores, so the ternary will always go through the second expression.